### PR TITLE
JFG-832: add PercentileAggregatorProvider & move Min/Max aggregator providers from jagger-test to jagger

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/MaxMetricAggregatorProvider.java [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-import java.util.ArrayList;
-
-/**
- * Created with IntelliJ IDEA.
- * User: kgribov
- * Date: 10/21/13
- * Time: 7:04 PM
- * To change this template use File | Settings | File Templates.
- */
-// begin: following section is used for docu generation - custom aggregator source
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/MaxMetricAggregatorProvider.java]: please, return comments for docu generation

File: test/jagger-test/src/main/resources/configuration/master/metric/calculatorsAndAggregators.conf.xml [hunk 2]
```diff
@@ -34,10 +34,8 @@
 
     <bean id="sin" class="metric.SinMetric"/>
 
-    <!--begin: following section is used for docu generation - custom aggregator-->
```
Comment (dlatnikov) [inline test/jagger-test/src/main/resources/configuration/master/metric/calculatorsAndAggregators.conf.xml]: please, return comment

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/PercentileAggregatorProvider.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        @Override
+        public void append(Number calculated) {
+            statisticsCalculator.addValue(calculated.doubleValue());
+        }
+
+        @Override
+        public Number getAggregated() {
+            Double result = statisticsCalculator.getPercentile(percentile);
+            return result.isNaN() ? null : result;
```
Comment (anton-antonenko) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/PercentileAggregatorProvider.java]: I've got this code from ticket description, so I don't know why it's null.

Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/collector/PercentileAggregatorProvider.java]: Ok. lets hope it is reasonable)

**Pull request merged**